onnxruntime/contrib_ops/cpu_contrib_kernels.cc

```diff
@@ -21,6 +21,8 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1,
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, Pad);
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, Unique);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, float, ConvTransposeWithDynamicPads);
+class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, float, CropAndResize);
+class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, double, CropAndResize);
```

Comment:
Curious - Need both float and double support ?

Reply:
RoiAlign supports both [here](https://github.com/microsoft/onnxruntime/blob/92dc5c506dbf2cc35df598d018070c29c90511a3/onnxruntime/core/providers/cpu/cpu_execution_provider.cc#L275-L276), I just follow it..

Reply:
I see.

The only concern is that this might bloat up binary size a bit since the implementation is templated and the release binary builds are with contrib ops included. I think you can drop `double` type if this is not needed to unblock model conversion (`double` will remain unsupported for this op just like `float16` will be unsupported)

Reply:
Yeah, feel free to drop double if not required.

Reply:
drop double now.